cli-output/src/cli_output.rs

```diff
@@ -3541,13 +3541,42 @@ mod tests {
             recent_timestamp: BlockTimestamp::default(),
             ..CliVoteAccount::default()
         };
+        #[rustfmt::skip]
+        let expected_output_common =
+            "Account Balance: 0.00001 SOL\n\
+             Validator Identity: 11111111111111111111111111111111\n\
+             Vote Authority: None\n\
+             Withdraw Authority: \n\
+             Credits: 0\n\
+             Commission: 0%\n\
+             Root Slot: ~\n\
+             Recent Timestamp: 1970-01-01T00:00:00Z from slot 0\n";
+
         let s = format!("{c}");
-        assert_eq!(s, "Account Balance: 0.00001 SOL\nValidator Identity: 11111111111111111111111111111111\nVote Authority: None\nWithdraw Authority: \nCredits: 0\nCommission: 0%\nRoot Slot: ~\nRecent Timestamp: 1970-01-01T00:00:00Z from slot 0\nEpoch Rewards:\n  Epoch   Reward Slot  Time                        Amount              New Balance         Percent Change             APR  Commission\n  1       100          1970-01-01 00:00:00 UTC  ◎0.00000001         ◎0.0000001                 11.000%          10.00%          1%\n  2       200          1970-01-12 13:46:40 UTC  ◎0.000000012        ◎0.0000001                 11.000%          13.00%          1%\n");
+        #[rustfmt::skip]
+        let expected_epoch_rewards_output =
+            "Epoch Rewards:\n  \
+             Epoch   Reward Slot  Time                        Amount              New Balance         Percent Change             APR  Commission\n  \
+             1       100          1970-01-01 00:00:00 UTC  ◎0.00000001         ◎0.0000001                 11.000%          10.00%          1%\n  \
+             2       200          1970-01-12 13:46:40 UTC  ◎0.000000012        ◎0.0000001                 11.000%          13.00%          1%\n";
+        assert_eq!(
+            s,
+            format!("{expected_output_common}{expected_epoch_rewards_output}")
+        );
         println!("{s}");
 
         c.use_csv = true;
         let s = format!("{c}");
-        assert_eq!(s, "Account Balance: 0.00001 SOL\nValidator Identity: 11111111111111111111111111111111\nVote Authority: None\nWithdraw Authority: \nCredits: 0\nCommission: 0%\nRoot Slot: ~\nRecent Timestamp: 1970-01-01T00:00:00Z from slot 0\nEpoch Rewards:\nEpoch,Reward Slot,Time,Amount,New Balance,Percent Change,APR,Commission\n1,100,1970-01-01 00:00:00 UTC,0.00000001,0.0000001,11%,10.00%,1%\n2,200,1970-01-12 13:46:40 UTC,0.000000012,0.0000001,11%,13.00%,1%\n");
+        #[rustfmt::skip]
+        let expected_epoch_rewards_output =
+            "Epoch Rewards:\n\
+             Epoch,Reward Slot,Time,Amount,New Balance,Percent Change,APR,Commission\n\
+             1,100,1970-01-01 00:00:00 UTC,0.00000001,0.0000001,11%,10.00%,1%\n\
+             2,200,1970-01-12 13:46:40 UTC,0.000000012,0.0000001,11%,13.00%,1%\n";
+        assert_eq!(
+            s,
+            format!("{expected_output_common}{expected_epoch_rewards_output}")
+        );
```

Comment:
`cargo fmt` didn't pick this one up automatically for some reason, happened to notice it because of the ugly string just above it.

I cleaned up this test in 3 commits starting with https://github.com/anza-xyz/agave/pull/7089/commits/af0c37c7547c86dd02c48ad94c6f30d9ceb0f345; the papertrail is hopefully easy to follow but I'll also call out that the assert statement is unchanged so I feel confident that my shifting around leaves the "rendered" string literal identical to how it was